dashboards/client-go.libsonnet

```diff
@@ -0,0 +1,161 @@
+
+      local rateLimiterDuration =
+        graphPanel.new(
+          'Rate Limiter duration 99th quantile',
+          datasource='$datasource',
+          format='s',
+          legend_show=true,
+          legend_values=true,
+          legend_current=true,
+          legend_alignAsTable=true,
+          legend_rightSide=true,
+        )
+        .addTarget(prometheus.target('histogram_quantile(0.99, sum(rate(rest_client_rate_limiter_duration_seconds_bucket{%(clusterLabel)s="$cluster", host=~"$host", verb=~"$verb"}[%(grafanaIntervalVar)s])) by (verb, host, le))' % $._config, legendFormat='{{verb}} {{host}}'));
+
+      local requestSize =
+        graphPanel.new(
+          'Request size 99th quantile',
+          datasource='$datasource',
+          format='s',
+          legend_show=true,
+          legend_values=true,
+          legend_current=true,
+          legend_alignAsTable=true,
+          legend_rightSide=true,
+        )
+        .addTarget(prometheus.target('histogram_quantile(0.99, sum(rate(rest_client_request_size_bytes_bucket{%(clusterLabel)s="$cluster", host=~"$host", verb=~"$verb"}[%(grafanaIntervalVar)s])) by (verb, host, le))' % $._config, legendFormat='{{verb}} {{host}}'));
+
+      local responseSize =
+        graphPanel.new(
+          'Response size 99th quantile',
+          datasource='$datasource',
+          format='s',
+          legend_show=true,
+          legend_values=true,
+          legend_current=true,
+          legend_alignAsTable=true,
+          legend_rightSide=true,
+        )
+        .addTarget(prometheus.target('histogram_quantile(0.99, sum(rate(rest_client_response_size_bytes{%(clusterLabel)s="$cluster", host=~"$host", verb=~"$verb"}[%(grafanaIntervalVar)s])) by (verb, host, le))' % $._config, legendFormat='{{verb}} {{host}}'));
+
+      dashboard.new(
+        '%(dashboardNamePrefix)sClient-go' % $._config.grafanaK8s,
+        time_from='now-1h',
+        uid=($._config.grafanaDashboardIDs['client-go.json']),
+        tags=($._config.grafanaK8s.dashboardTags),
+      ).addTemplate(
+        {
+          current: {
+            text: 'default',
+            value: $._config.datasourceName,
+          },
+          hide: 0,
+          label: 'Data Source',
+          name: 'datasource',
+          options: [],
+          query: 'prometheus',
+          refresh: 1,
+          regex: $._config.datasourceFilterRegex,
+          type: 'datasource',
+        },
+      )
+      .addTemplate(
+        template.new(
+          'host',
+          '$datasource',
+          'label_values(%(clusterLabel)s="$cluster", host)' % $._config,
+          label='host',
+          refresh='time',
+          includeAll=true,
+          sort=1,
+        )
+      )
+      .addTemplate(
+        template.new(
+          'verb',
+          '$datasource',
+          'label_values(%(clusterLabel)s="$cluster", verb)' % $._config,
```

Comment:
This is converted to a promQL query that will be executed by Grafana and in your case, `label_values(%(clusterLabel)s="$cluster", verb)` is getting converted to: `label_values(cluster=\"$cluster\", verb)` which is not a valid query since your are not selecting any metric. Considering the labels that you are selecting, I suppose you want the following:
```suggestion
          'label_values(rest_client_requests_total{%(clusterLabel)s="$cluster"}, verb)' % $._config,
```